book.js

```diff
@@ -1,14 +1,50 @@
 
 
+let admin_log_out_btn=document.querySelector("#admin-log-out-btn");
+
+admin_log_out_btn.addEventListener("click",()=>
+    {
+     
+        window.location.href = "index.html"; 
+        
+    })
+
+ let goto_dashboard_btn=document.querySelector("#goto-dashboard");
+
+    goto_dashboard_btn.addEventListener("click",()=>
+        {
+            window.location.href = "admin_dashboard.html"; 
+        })
+
+
+
+
+
+
+
+
+
+
 const books = [
   { name: "OOP", category: "programming", price: 30, author: "Robert C. Martin", isbn: "123" },
   { name: "python", category: "AI", price: 36, author: "Stuart Russell", isbn: "234" },
   { name: "React", category: "programming", price: 28, author: "Kirupa Chinnathambi", isbn: "890" }
 ];
 
 
-function createElement_of_form()
-{
+
+
+
+function openForm(bookToUpdate) {
```

Comment:
function name should be descriptive as `openForm` is a vague name. This should be updated with `openUpdateForm`